app/controllers/mixins/ems_common_angular.rb

```diff
@@ -110,6 +112,16 @@ def create_ems_button_validate
       update_ems_button_validate(ems)
     end
 
+    def add_proxy_custome_attributes(ems, params)
+      %w(no_proxy http_proxy https_proxy).each do |att|
+        key = "container_" + att
+        ems.custom_attributes.create(
+          :name  => att.upcase,
+          :value => params[key]
```

Comment:
@enoodle we're not dropping it, we're just moving it in another PR so that we can get this done quicker. Also it will help you to separate better the business logic from the UI.